### PR TITLE
Paddle split test fp32 int8

### PR DESCRIPTION
**Pull request by pawelpiotrowicz**
**Add two extra flags for test_analyzer_int8_image_classification to disable fp32/int8 .**

paddle-public/build/paddle/fluid/inference/tests/api/test_analyzer_int8_image_classification "ARGS" "--infer_model=/home/pawepiot/workspace/multi_instance_public/paddle-public/build/third_party/inference_demo/int8v2/resnet50/model" "--infer_data=/home/pawepiot/workspace/multi_instance_public/paddle-public/build/third_party/inference_demo/int8v2/data.bin" "--warmup_batch_size=100" "--batch_size=50" "--paddle_num_threads=4" "--iterations=2" "--**enable_int8=false"**

paddle-public/build/paddle/fluid/inference/tests/api/test_analyzer_int8_image_classification "ARGS" "--infer_model=/home/pawepiot/workspace/multi_instance_public/paddle-public/build/third_party/inference_demo/int8v2/resnet50/model" "--infer_data=/home/pawepiot/workspace/multi_instance_public/paddle-public/build/third_party/inference_demo/int8v2/data.bin" "--warmup_batch_size=100" "--batch_size=50" "--paddle_num_threads=4" "--iterations=2" "--**enable_fp32=false**"

### CODE REVIEW

File: paddle/fluid/inference/tests/api/tester_helper.h [hunk 1]
```diff
@@ -463,68 +465,90 @@ void SummarizeAccuracy(float avg_acc_fp32, float avg_acc_int8,
...
+  float total_accs{0};
+
+  for (size_t i = 0; i < output_slots.size(); ++i) {
+    switch (compared_idx) {
+      case 1:
+        PADDLE_ENFORCE_GE(
+            output_slots[i].size(), 2UL,
+            "To achieve top 1 accuracy, output_slots_quant[i].size()>=2");
+        break;
+
```
Comment (mozga-intel) [inline paddle/fluid/inference/tests/api/tester_helper.h]: Could you please remove the following white-line?

**Review by mozga-intel — DISMISSED** [review]
LGTM!

File: paddle/fluid/inference/tests/api/tester_helper.h (hunk 1 above)
Comment (pawelpiotrowicz) [reply paddle/fluid/inference/tests/api/tester_helper.h]: done

**Comment by luotao1** [comment]
please use `test=develop` to trigger the CI

**Review by luotao1 — APPROVED** [review]

**Pull request merged**